server/src/Controller/User/UserController.js

```diff
@@ -38,8 +38,18 @@ const getAllUsers = async (req, res, next) => {
 
 const createUser = async (req, res, next) => {
   try {
-    const { name, email, password } = req.body;
-    const data = { name, email, password };
+    const {
+      name,
+      email,
+      password,
+      bio,
+    } = req.body;
+    const data = {
+      name,
+      email,
+      password,
+      bio,
+    };
     const result = await create(userModel, data);
```

Comment:
existence of user can be checked before creating

Reply:
Different Validations are still to be added. There is a different issue for that.